gengokit/template/NAME-service/generated/endpoints.gotemplate

```diff
@@ -59,3 +59,13 @@ type Endpoints struct {
 		}
 	{{end}}
 {{end}}
+
+// WrapAll wraps each Endpoint field of struct Endpoints with a
+// go-kit/kit/endpoint.Middleware
```

Comment:
Add clarifying sentence like `Use this for applying a set of middlewares to every endpoint in the service. The middlewares will be applied in the order passed, with the first middleware being the outermost middleware.`